include/swift/Basic/Statistic.h

```diff
@@ -13,7 +13,10 @@
 #ifndef SWIFT_BASIC_STATISTIC_H
 #define SWIFT_BASIC_STATISTIC_H
 
+#include "llvm/ADT/SmallString.h"
 #include "llvm/ADT/Statistic.h"
+#include "swift/Basic/Timer.h"
+#include "swift/FrontendTool/FrontendTool.h"
```

Comment:
Eh, you know, if I am going that way .. I didn't wind up relying on more than 2 methods in FrontendObserver and they're both called within FrontendTool.cpp I can avoid routing through it and just call two local, explicit count-the-things functions in FrontendTool.cpp.